### PR TITLE
LCH-3485: Add functions to handle webhook suppression.

### PR DESCRIPTION
**Pull request by Ateee89**

### CODE REVIEW

File: src/ContentHubClient.php [hunk 1]
```diff
@@ -787,6 +787,30 @@ public function updateWebhook($uuid, array $options) {
...
+   * Remove suppression from webhook.
+   *
+   * @param string $webhook_uuid
+   *   Webhook uuid.
+   *
+   * @return \Psr\Http\Message\ResponseInterface
+   */
+  public function unSuppressWebhook(string $webhook_uuid) {
+    return $this->delete("webhook/$webhook_uuid/suppress");
+  }
```
Comment (c0d39uru) [inline src/ContentHubClient.php]: For the record, this is not the right way of interacting with an API as a client library. Client library (in this case, Content Hub PHP, is supposed to not only talk to the API, but also take care of all the possible results including success and failures. 
I know there are other functions in this library that does the exact same thing (simply returning the raw http call), but it's legacy. Now that we are adding a new one, we should do it the right way.

Keep in mind, the library's consumer should 100% rely on the library for error checking and handling. The purpose of he library is not to make the HTTP call only.

**Review by c0d39uru — APPROVED** [review]

**Comment by c0d39uru** [comment]
@Ateee89 Any idea why coveralls is reporting coverage decrease?

**Comment by EclipseGc** [comment]
Can we get test coverage for these two new methods?

**Comment by Ateee89** [comment]
@EclipseGc @cleancode4ever Sorry totally forgot about the coverage, fixed now.

**Pull request merged**